cmd/vault-secret-collection-manager/README.md

```diff
@@ -25,3 +27,10 @@ Endpoints:
 * `GET /secretcollection`: Returns a list of all secret collections for the current user
 * `PUT /secretcollection/:name`: Creates a new secret collection using the provided `name`. The secret collection must not exist yet.
 * `PATCH /secretcollection/:name`: Changes the members of an existing secret colltion. The requesting user must be a member of the collection.
+
+## Development
+
+* Use `docker-compose` to start, vault, an oauth2 proxy and a dex instance as an IDP: `cd cmd/vault-secret-collection-manager && docker-compose up`
```

Comment:
```
$ docker-compose up
bash: docker-compose: command not found...
```

:(

Any nontrivial way to get this working on normal CentOS/Fedora?

Reply:
https://github.com/containers/podman-compose